src/coreclr/jit/emitxarch.cpp

```diff
@@ -7440,6 +7438,30 @@ size_t emitter::emitSizeOfInsDsc(instrDesc* id)
                     return sizeof(instrDesc);
                 }
             }
+        case ID_OP_AMD:
+        case ID_OP_AMD_CNS:
+            if (id->idIsLargeCns())
```

Comment:
Based on `emitNewInstrAmdCns`, I would expect it to be:
```
            if (id->idIsLargeCns())
            {
                if (id->idIsLargeDsp())
                {
                    return sizeof(instrDescCnsAmd);
                }
                else
                {
                    return sizeof(instrDescCns);
                }
            }
            else
            {
                if (id->idIsLargeDsp())
                {
                    return sizeof(instrDescAmd);
                }
                else
                {
                    return sizeof(instrDesc);
                }
            }
```

Reply:
Actually yes.